### DISCUSSION TITLE
CI service scheduled maintenance - 20 February 2023

### DISCUSSION THREAD

**Discussion by jcar87** [conan-io/conan-center-index#16176] [Announcements]
Our CI service for Conan Center Index Pull Requests will be down for maintenance during 20-22 Feburary 2023 due to upgrades needed for the upcoming Conan 2.0 release.

During the downtime, newly opened or updated pull requests will not be processed by our CI service. Some Pull Requests may be manually built by the Conan Center team in order to test new functionality.

New pull requests can be opened but they will be built only after the service is resumed. Please also consider making sure your recipe is ready for Conan 2.0.

For further information, please read:
- ConanCenterIndex - Conan v2 Roadmap: https://github.com/conan-io/conan-center-index/blob/master/docs/v2_roadmap.md
- Conan migration guide to 2.0: https://docs.conan.io/en/latest/conan_v2.html

For service status updates, please check https://status.conan.io/

**Comment by jiaoew1991** [comment]
Hi @jcar87, When will the system maintenance end?

**Reply by jcar87** [reply]
Hi @jiaoew1991 - we're hoping to resume the CI service before the end of the working week. We continue testing functionality related to the Conan 2.0 release. 

However we should be good to run some Pull Requests ourselves - if there is anything you'd like us to prioritise, please let us know :) 

Thanks!

**Reply by jiaoew1991** [reply]
Thx @jcar87 , I want to run https://github.com/conan-io/conan-center-index/pull/16220, my project is blocked by the compile error 😅

**Comment by uilianries** [comment]
The ConanCenterIndex CI will be restored soon.

Please, avoid closing/opening your active pull requests on ConanCenterIndex, it may overflow the build queue. Also, re-opening a PR will push it to the end of the build list. The CI is prepared to digest all PRs which are waiting since last week.

Also, please, avoid opening new PRs next hours in case they are only minors and may not fix an important/urgent bug. This will contribute on not breaking the build system, as we are expecting more builds than usual on next few days.

Thank you for your patience!

**Comment by uilianries** [comment]
The maintenance has been finished and ConanCenterIndex is back on rails!

Next hours will be watched closely to validate the current state.

Some few packages on ConanCenterIndex can be missing for Conan 2.0. For this kind of situation, the package will be generated internally, when possible.

In case you find anything unusual, please, open an issue to https://github.com/conan-io/conan-center-index/issues

**Comment by patmantru** [comment]
Will "only a version bump" PRs be processed automatically, or will they need to be approved by humans?
https://github.com/conan-io/conan-center-index/pull/16218

THX!!!

**Reply by uilianries** [reply]
No human review is required when only doing a Bump version: https://github.com/conan-io/conan-center-index/blob/master/docs/labels.md#bump-version

**Reply by patmantru** [reply]
Hmm...I used the updateable-recipies bot output to create the PR, it's only a version bump, but it doesn't have the `bump-version` label on it.
